bin/reflective_loader/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+[package]
+name = "reflective_loader"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.dev]
```

Comment:
`cargo test` has issues when these aren't set.
Seems like a linker issue.

Reply:
Usually I implement all features in a lib crate and write the tests against that and just call a single lib entrypoint from the cdylib. 
Up to you, this works fine.

Reply:
I believe that's what i'm doing.
The `./bin/reflective_loader` being the lib crate and eldritch calling the entry-point through `CreateRemoteThread`.
Or are you thinking something different that that? ^